droptools-example/src/main/java/com/bendb/example/resources/PostsResource.java

```diff
@@ -25,35 +28,38 @@
 @Path("posts")
 @Produces(MediaType.APPLICATION_JSON)
 public class PostsResource {
+
     @GET
-    public List<BlogPost> findPostsByTag(@QueryParam("tag") String tag, @Context DSLContext create) {
+    public List<BlogPost> findPostsByTag(@QueryParam("tag") String tag, @JooqInject @Instance("slave") DSLContext slave) {
```

Comment:
Out of curiosity what do you think is better:

@JooqInject @Instance("slave") DSLContext slave

or

@JooqInject("slave") DSLContext slave